docs/root/version_history/current.rst

```diff
@@ -113,6 +113,7 @@ New Features
 * jwt_authn: added support for :ref:`Jwt Cache <envoy_v3_api_field_extensions.filters.http.jwt_authn.v3.JwtProvider.jwt_cache_config>` and its size can be specified by :ref:`jwt_cache_size <envoy_v3_api_field_extensions.filters.http.jwt_authn.v3.JwtCacheConfig.jwt_cache_size>`.
 * jwt_authn: added support for extracting JWTs from request cookies using :ref:`from_cookies <envoy_v3_api_field_extensions.filters.http.jwt_authn.v3.JwtProvider.from_cookies>`.
 * listener: new listener metric ``downstream_cx_transport_socket_connect_timeout`` to track transport socket timeouts.
+* lua: added ``header:getAtIndex()`` and ``header:getValueSize()`` methods to get the value of a header at certain index and get the value size of a given header.
```

Comment:
Nit. Add ref to the `config_http_filters_lua_header_wrapper` that we have just moved.

Probably something like: ...methods to :ref:`header object <config_http_filters_lua_header_wrapper>` ...

Reply:
Added, Thanks!